aio/content/examples/testing/src/app/banner/banner-external.component.spec.ts

```diff
@@ -37,18 +35,15 @@ describe('BannerComponent (external files)', () => {
 
   describe('One beforeEach', () => {
     // #docregion one-before-each
-    beforeEach(waitForAsync(() => {
-      TestBed
-          .configureTestingModule({
-            declarations: [BannerComponent],
-          })
-          .compileComponents()
-          .then(() => {
-            fixture = TestBed.createComponent(BannerComponent);
-            component = fixture.componentInstance;
-            h1 = fixture.nativeElement.querySelector('h1');
-          });
-    }));
+    beforeEach(async () => {
+      await TestBed.configureTestingModule({
+        declarations: [BannerComponent],
+      })
+      .compileComponents();
```

Comment:
May as well join up these lines:

```suggestion
      }).compileComponents();
```